airflow/contrib/plugins/metastore_browser/main.py

```diff
@@ -86,7 +87,7 @@ def db(self):
         return self.render(
             "metastore_browser/db.html", tables=tables, db=db)
 
-    @wwwutils.gzipped
```

Comment:
This change seems unrelated - we're still importing wwwutils...?

Reply:
in the rbac version the `gzipped` decorator is not in the utils.py anymore, instead it was moved to the decorators.py

Reply:
Ah!